utils/wasm-gen/src/lib.rs

```diff
@@ -135,6 +138,7 @@ impl GearConfig {
             skip_init: (1, 1000).into(),
             skip_handle: prob,
             skip_init_when_no_funcs: prob,
+            remove_recursion: (95, 100).into(),
```

Comment:
Isn't is better to add a const?

Reply:
It makes sense however I'm not sure about the case. Frankly speaking the value is also a subject to discuss